imagetest/test_suites/image_validation/linux_license_test.go

```diff
@@ -0,0 +1,64 @@
+package imagevalidation
+
+import (
+	"strings"
+	"testing"
+	"unicode"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+var imageLicenseCodeMap = map[string]string{
```

Comment:
I haven't test all the license code. I check the license API and map them based on the license name

Reply:
it should be easy to run this test against all these images and see if they all succeed

Reply:
<testsuite name="image_validation-ubuntu-minimal-2004-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-ubuntu-2004-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.01"></testcase>
	</testsuite>
	<testsuite name="image_validation-ubuntu-1604-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.01"></testcase>
	</testsuite>
	<testsuite name="image_validation-ubuntu-1804-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-ubuntu-minimal-1804-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-7-4-sap" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.01"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-7-6-sap-ha" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.01"></testcase>
	</testsuite>
	<testsuite name="image_validation-ubuntu-minimal-1604-lts" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-7-7-sap-ha" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-8-2-sap-ha" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-7" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.01"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-8" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-centos-8" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-centos-7" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-debian-9" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
	</testsuite>
	<testsuite name="image_validation-rhel-8-1-sap-ha" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
		<testcase classname="" name="TestLinuxLicense" time="0.02"></testcase>
	</testsuite>
	<testsuite name="image_validation-debian-10" tests="1" failures="0" errors="0" disabled="0" skipped="0" time="0">
	<testcase classname="" name="TestLinuxLicense" time="0"></testcase>
</testsuites>

Reply:
Add test result in comment